intel_extension_for_transformers/neural_chat/examples/serving/TGI/tgi.yaml

```diff
@@ -29,6 +29,10 @@ device: "auto"
 serving:
     framework: "tgi"
     tgi_engine_params:
+        # when endpoint is set, neuralchat will not start a tgi service,
+        # and other params will not work
+        endpoint: "http://0.0.0.0:9876/"
```

Comment:
if endpoint has the port, why we need port again?

Reply:
It's two different ways to launch TGI service with neuralchat.
- By endpoint: User already started a TGI service on their local server with a complete endpoint like "http://0.0.0.0:9876/". Neuralchat will consume this endpoint directly, without starting a TGI service.
- By port: Neuralchat will help user to start a TGI service automatically with this port, and consume the service.
So the `endpoint` and `port` will not exist at the same time. Just place here to show the usability.